web_testing_go/MODULE.bazel

```diff
@@ -1,10 +1,11 @@
 module(
     name = "rules_web_testing_go",
-    version = "0.4.0",
+    version = "0.4.1",
 )
 
 bazel_dep(
     name = "rules_webtesting",
+    version = "0.4.0",
```

Comment:
We should pin `rules_webtesting` to `0.4.1` in all the submodules. Otherwise we'll always be in a weird state where the submodules are one version ahead of the parent.

Reply:
They'll all come into existence at the same time, and since we use the `local_path_override`, that will take precedence during presubmit. Looking at rules_rust releases, they seem to be [bumping all versions in one shot](https://github.com/bazelbuild/rules_rust/commit/646b9a5bdaeb68e40a3ec3ae47aa4aeb1b38a508).